DWEB_ADDRESSING.md

```diff
@@ -0,0 +1,40 @@
+# ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square) Addressing on the Decentralized Web
+
+**Maintainer(s)**:
+- [Lars Gierth](mailto:lgierth@ipfs.io)
```

Comment:
Maybe include @lidel here since he has lead this work in browsers?

Reply:
I didn't make any decisions on who maintains what for this PR, only focused or reorg & cleanup of the files + writing the maintainer protocol. Let's leave the election/volunteering of spec maintainers for an iteration after merging the protocol (so that expectations are clear from the start).